src/protocol/mod.rs

```diff
@@ -1,3 +1,6 @@
+pub mod context;
+pub mod securemul;
```

Comment:
ditto: lets just export `SecureMul` struct so code can import it as `use crate::protocol::SecureMul`